package.json

```
@@ -81,6 +81,7 @@
     "pg": "^7.12.1",
```

```suggestion
"pg": "^8.7.1"
```
This change will allow the app to run on my machine without any changes.